scripts/gulpfiles/typings.js

```diff
@@ -108,10 +108,14 @@ function typings() {
 function msgTypings(cb) {
   const template = fs.readFileSync(path.join('typings/templates/msg.template'), 'utf-8');
   const msgFiles = fs.readdirSync(path.join('msg', 'json'));
+  const msgDir = path.join(TYPINGS_BUILD_DIR, 'msg');
+  if (!fs.existsSync(msgDir)) {
+    fs.mkdirSync(msgDir, {recursive: true});
+  }
   msgFiles.forEach(msg => {
     const localeName = msg.substring(0, msg.indexOf('.json'));
     const msgTypings = template.slice().replace(/<%= locale %>/gi, localeName);
-    fs.writeFileSync(path.join(BUILD_DIR, 'msg', localeName + '.d.ts'), msgTypings, 'utf-8');
+    fs.writeFileSync(path.join(TYPINGS_BUILD_DIR, 'msg', localeName + '.d.ts'), msgTypings, 'utf-8');
```

Comment:
There are many overly-long lines in the gulpfiles, but might as well fix this one while you're here.

Reply:
@googlebot I consent.